VideoLocker/res/values/styles.xml

```diff
@@ -106,6 +105,10 @@
         <item name="android:background">@color/edx_brand_primary_base</item>
     </style>
 
+    <style name="course_dashboard_separator_style" parent="separator_style">
```

Comment:
that'll make their name quite big like `edx_grayscale_neutral_light_separator_style`. Do we want that ?